updateHostsFile.py

```diff
@@ -943,14 +966,6 @@ def remove_dups_and_excl(merge_file, exclusion_regexes, output_file=None):
                 if line and not line.startswith("#"):
                     settings["exclusions"].append(line)
 
-    if not os.path.exists(settings["outputpath"]):
```

Comment:
I changed output_file to required, and I made a few changes here, the file is already ensure created by `remove_old_hosts_file`